### PR TITLE
When looking for audio files, follow symlinks by default.

### PR DESCRIPTION
**Pull request by iver56**
Relates to https://github.com/asteroid-team/torch-audiomentations/commit/01f1436481c3b8f67a30caf7e0578a183044ef0f

@hbredin

### CODE REVIEW

File: .github/workflows/test_requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 numpy>=1.14,<1.18
 scipy>=1.5.2
-torch>=1.6.0
+torch==1.7.0
```
Comment (hbredin) [inline .github/workflows/test_requirements.txt:3]: Was this change intended?

**Review by hbredin — APPROVED** [review]
LGTM. Would have enjoyed a few comments in the unit test though 👍

File: .github/workflows/test_requirements.txt (hunk 1 above)
Comment (iver56) [reply .github/workflows/test_requirements.txt:3]: Not sure. But the symptom was that the CI did not successfully run. I think it was due to torch 1.7.1 (the 700 MB variant) being download instead of the 1.7.0 CPU one (~60 MB), which we ran in tests earlier.
